media/js/base/mozilla-fxa-link.js

```diff
@@ -57,6 +93,25 @@ if (typeof window.Mozilla === 'undefined') {
                 link.setAttribute('data-mozillaonline-link', FxaLink.updateURL(mozillaOnlineLink));
             }
         }
+
+        if (useUITourForFxA) {
+            Mozilla.UITour.ping(function() {
+                for (var i = 0; i < fxaSigninLink.length; i++) {
```

Comment:
We need to make sure that this only intercepts links to accounts.firefox.com, and not links to other fxa related flows, such as monitor.firefox.com. I'd suggest adding an allowed list of domains, similar to how we do here: https://github.com/mozilla/bedrock/blob/master/media/js/base/mozilla-fxa-product-button.js#L15-L21

Reply:
Also, do we need to add in extra logic to this change to handle the China repack for accounts.firefox.com.cn?

Reply:
For the China repack, the UITour should intercept it and then hand it off to the client and that would use the FxA configuration that is on the client. I think if the user has the client configured for China servers then the client would send them to those servers.